src/fides/api/schemas/connection_configuration/connection_secrets_scylla.py

```diff
@@ -0,0 +1,46 @@
+from typing import List, Optional
+
+from pydantic import Field
+
+from fides.api.schemas.base_class import NoValidationSchema
+from fides.api.schemas.connection_configuration.connection_secrets import (
+    ConnectionConfigSecretsSchema,
+)
+
+
+class ScyllaSchema(ConnectionConfigSecretsSchema):
+    """Schema to validate the secrets needed to connect to a Scylla Database"""
+
+    host: str = Field(
+        title="Host",
+        description="The hostname or IP address of the server where the database is running.",
+    )
+    port: int = Field(
+        9042,
+        title="Port",
+        description="The network port number on which the server is listening for incoming connections (default: 9042).",
+    )
+    username: str = Field(
+        title="Username",
+        description="The user account used to authenticate and access the database.",
+    )
+    password: str = Field(
+        title="Password",
+        description="The password used to authenticate and access the database.",
+        sensitive=True,
+    )
+    keyspace: Optional[str] = Field(
+        title="Keyspace",
+        description="The keyspace used.",
+        sensitive=True,
+    )
```

Comment:
Adding an optional keyspace here - might be useful to specify a specific one for DSR's, but this is not required for detection and discovery -

Reply:
interesting, i'm not sure how we'll want to handle this on the d&d side - ultimately i think we'll want to be able to leverage the same connection config for both DSRs and d&d monitors - so maybe it's just something that users will have "to know" that this won't impact the d&d workflow.

not worth any changes for now, i think how you've handled it is fine - just pondering...